### ISSUE TITLE
[🐛 Bug]:  Mounted Volume Permissions Issues

### ISSUE THREAD

**Issue by Ducky6944** [SeleniumHQ/docker-selenium#1477]
### What happened?

Using the directions from this [issue](https://github.com/SeleniumHQ/docker-selenium/issues/1095#issuecomment-674671360) I created a user, group, and folder to mount on host.

```BASH
> sudo groupadd -g 1201 seluser
> sudo useradd -g seluser -u 1200 -M -N seluser
> sudo usermod -L seluser # revoke the ability to login
> mkdir selenium
> sudo chown seluser:seluser selenium
> ls -la 
rwxrwxr-x   3   myuser   myuser      4 KiB   Sat Jan  8 04:30:36 2022   ✓     ./
rwxrwxr-x   7   myuser   myuser      4 KiB   Sat Jan  8 02:38:44 2022   ✓     ../
rwxrwxr-x   2   seluser  seluser     4 KiB   Sat Jan  8 04:30:36 2022         selenium/
> id -u seluser
1200
> id -g seluser
1201
```
I then setup a docker compose which pretty much verbatim to the template in the repo. I had to change some of the ports and changed the container names. 

```YAML
version: "3"
services:

  firefox:
    container_name: work_selenium_firefox
    image: selenium/node-firefox:4.1.1-20211217
    shm_size: 2gb
    depends_on:
      - selenium-hub
    environment:
      - SE_EVENT_BUS_HOST=selenium-hub
      - SE_EVENT_BUS_PUBLISH_PORT=4442
      - SE_EVENT_BUS_SUBSCRIBE_PORT=4443
    ports:
      - "17003:7900"
    volumes:
      - ~/docker/.storage/selenium:/home/seluser/Downloads

  selenium-hub:
    container_name: work_selenium_hub
    image: selenium/hub:4.1.1-20211217
    ports:
      - "17000:4442"
      - "17001:4443"
      - "17002:4444"

```
I then went started the stack and wen to check permissions
```BASH
> sudo docker-compose up -d
> sudo docker exec -it work_selenium_firefox /bin/bash
> cd /home/seluser
> ls -la 
total 56
drwxrwxrwx 1 seluser root    4096 Jan  8 04:31 .
drwxr-xr-x 1 root    root    4096 Dec 17 22:34 ..
-rwxrwxrwx 1 seluser root     220 Feb 25  2020 .bash_logout
-rwxrwxrwx 1 seluser root    3771 Feb 25  2020 .bashrc
drwxr-xr-x 4 seluser seluser 4096 Jan  8 04:31 .cache
drwx------ 3 seluser seluser 4096 Jan  8 04:31 .config
drwxr-xr-x 2 root    root    4096 Jan  8 03:13 Downloads
-rwxr-xr-- 1 seluser seluser   83 Jan  8 04:31 .fehbg
drwx------ 2 seluser seluser 4096 Jan  8 04:31 .fluxbox
drwx------ 3 seluser seluser 4096 Jan  8 04:31 .mozilla
-rwxrwxrwx 1 seluser root     807 Feb 25  2020 .profile
-rwxrwxrwx 1 seluser root       0 Dec 17 22:38 .sudo_as_admin_successful
drwxrwxrwx 1 seluser root    4096 Dec 17 22:38 .vnc
-rwxrwxrwx 1 root    root     209 Dec 17 22:38 .wget-hsts
```
In addition I ran a selenium script which works in any seluser folder that is not a mounted volume. Saving into the mounted volume prompts me for a name because it is unable to write to the path. `chown 1200:1201 selenium` didn't seem to resolve the issue either. Lastly I tried setting permissions to 777 for the folder to no avail.

Platform
```
Ubuntu 20.10 Server
Docker version 20.10.12, build e91ed57
```

Any advice or guidance would be helpful. Thanks!

### Command used to start Selenium Grid with Docker

```shell
Listed above.
```

### Relevant log output

```shell
NA
```

### Operating System

Ubuntu 20.10

### Docker Selenium version (tag)

Docker version 20.10.12, build e91ed57

**Comment by diemol** [comment]
Please follow the instructions shown here https://github.com/SeleniumHQ/docker-selenium#mounting-volumes-to-retrieve-downloaded-files

**Comment by Ducky6944** [comment]
> Please follow the instructions shown here https://github.com/SeleniumHQ/docker-selenium#mounting-volumes-to-retrieve-downloaded-files

I did try that at least 5-10 times, but it does work. I'm not sure if it was my attempt to try other workarounds first, or if I accidentally put a sudo on the wrong command eg mkdir for example, or if it was because I was using `~/` as a relative path in my compose file, but I found a folder created in /root call selenium as in my example above. I removed it and changed my compose to use a explicit path and it totally works. 

![the-simpsons-homer](https://user-images.githubusercontent.com/12100960/148637534-d678815f-1b7e-4dbf-9bc4-855d36922c47.gif)
